Tasks/SqlAzureDacpacDeployment/make.json

```diff
@@ -25,6 +25,13 @@
                     }
                 ]
             }
+        ],
+        "archivePackages": [
+            {
+                "archiveName": "MSDeploy.zip",
```

Comment:
You can update this file with values how it should look once uploaded and update PR.